src/include/sof/lib/notifier.h

```diff
@@ -45,7 +45,7 @@ struct notifier {
 	void (*cb)(int message, void *cb_data, void *event_data);
 };
 
-#if defined(CLK_SSP) && defined(NOTIFIER_ID_SSP_FREQ)
```

Comment:
The format for fixes is: Fixes: comitID ("Commit subject") where commit id has at least 12 chars from commit hash.

Reply:
I'll keep it to 10 because SOF will probably not end up with hundreds of thousands of commits. Updating to add message.

Reply:
Yes, that worksforme. For next patches please mind that you shouldnt add a new line between Fixes and Signed-off-by tags.

Reply:
Removed newline from this one too, thanks.